src/mono/wasm/testassets/WasmOnAspNetCore/WasmBrowserClient/SignalRTest.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices.JavaScript;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.AspNetCore.Http.Connections;
+
+public class SignalRTest
```

Comment:
This is same code (+ hub on server) as we have for Blazor, isn't it? 
Could we have a single test app containing
- AspNetCore server
- Blazor client
- Wasm client
- Shared lib for both client projects

Host both clients in the same server on different sub-paths. What I'm looking for is having a minimal set of test apps (because there is some maintenance with them) that would be understandable. In that terms connecting all signalr pieces together seems reasonable.

Reply:
I was not able to figure out the proper way of serving `blazor`, so only `wasmbrowser` is served from a subdir. Beyond that, it's possible that the app is in desired form already.